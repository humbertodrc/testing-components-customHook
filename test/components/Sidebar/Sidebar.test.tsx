import { render } from '@testing-library/react';
import React from 'react';
import Sidebar from '../../../src/components/Sidebar/Sidebar';

describe('Pruebas en <Sidebar />', () => {

  test('Debe de mostrarse correctamente', () => {
    const { container } = render(<Sidebar visible={false} close={() => { }} />);
    expect(container).toMatchSnapshot();
  });

})