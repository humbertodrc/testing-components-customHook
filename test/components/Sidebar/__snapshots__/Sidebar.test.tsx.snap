// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <Sidebar /> Debe de mostrarse correctamente 1`] = `<div />`;
